src/styles/_search.scss

```diff
@@ -1,20 +1,34 @@
 /***** Search field *****/
-
-// Homepage search
-
 .search {
   input[type='search'] {
     border: 1px solid var(--mm-color-border);
     border-radius: var(--mm-border-radius);
     color: var(--mm-color-primary-text);
+    height: 40px;
     width: 100%;
 
     &:focus {
-      border: var(--mm-outline);
+      box-shadow: 0 0 0 2px var(--mm-color-logo-blue-light);
+      outline: none;
     }
   }
 }
 
+.hero-inner .search,
+.header-search .search {
+  &::before {
+    background: no-repeat url('/assets/svg/icon-search.svg');
```

Comment:
Sometimes when I run `npm run develop` this file disappears from the `dist` folder. I'm not sure why this is the case.

Reply:
Wuff. I've been testing this out and cannot replicate. 😞 

In theory, as along as the file lives inside `src/assets`, it should be copied over any time the Zendesk server restarts. The Gulp task is asynchronous though, so just because the tasks are defined in a certain order, there can be outside circumstances that effect the order of operations. We are using two different machines with different levels of performance, so I can see this possibly being an issue. That'd also make sense as to why it happens "sometimes" for you. 

I'll pursue this path further.